### PR TITLE
added date

### PR DESCRIPTION
**Pull request by ianchen06**
# READY FOR REVIEW/NOT READY
- (Edit the above to reflect status)

# Summary
- TL;DR - what's this PR for?

# Needed By (Date)
- When does this need to be merged by?

# Criticality
- How critical is this PR on a 1-10 scale?
- E.g., it affects one site, or every site and product?

# Steps to Test

1. Do this
2. Then this
3. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Review by ianchentw — COMMENTED** [review]
I think this is good?

File: index.html [hunk 1]
```diff
@@ -8,5 +8,7 @@
   <body>
     <h1>Hello, world</h1>
     <p>add date</p>
+    <p>add date</p>
```
Comment (ianchentw) [inline index.html]: Is this necessary?

**Review by ianchentw — COMMENTED** [review]
okk

File: index.html (hunk 1 above)
Comment (ianchen06) [reply index.html]: got it...

**Comment by ianchentw** [comment]
LGTM
